packages/common/src/discussions/api/types.ts

```diff
@@ -817,13 +799,11 @@ export interface ICreateChannelPermissions {
  * @interface IUpdateChannelPermissions
  */
 export interface IUpdateChannelPermissions {
-  access?: SharingAccess;
-  allowAnonymous?: boolean;
-  groups?: string[];
+  channelAclDefinition?: IChannelAclPermissionDefinition[];
```

Comment:
Similar point as above, impacts Discussion Board participation pane where we create/edit channels. Need ACL UI versus legacy access/groups/allowAnonymous